design/authenticators/application-identity/application-identity.md

```diff
@@ -0,0 +1,464 @@
+# Application Identity
+
+##### Note
+
+In this doc I talk about authenticating hosts with Conjur. Everything I say
+ about authenticating hosts is relevant also for authenticating users
+ . Whenever I say one of `host`, `user` or `role` it has the same meaning.
+  
+## Introduction
+
+In several authenticators we require hosts to contain some data which define
```

Comment:
in several or in all? I would list them in (). Example: (authn-k8s, authn-azure, ...)